lib/grell/rawpage.rb

```diff
@@ -36,6 +35,17 @@ def has_selector?(selector)
       page.has_selector?(selector)
     end
 
+    def finished_all_ajax_requests?(timeout, interval)
+      Timeout::timeout(timeout) do
```

Comment:
It doesn't work without `Timeout` here.
For some reason `page.evaluate_script('jQuery.active')` never returns 0 if `Timeout` is removed